### PR TITLE
Trying rustfmt

### PR DESCRIPTION
**Pull request by fabricedesre**
Let's see if https://github.com/rust-lang-nursery/rustfmt works for us.

### CODE REVIEW

**Comment by fabricedesre** [comment]
The changes look huge but it's been all done by running `cargo fmt` in the main directory and in each component/ subdirectory.

I like that this gives us consistent style and formatting without effort, even if this doesn't always match my personal preferences.

If we go this route, we need to enforce that consistently, eg.  by running rustfmt like clippy in the pre-commit hook.

File: components/openzwave-adapter/src/lib.rs [hunk 1]
```diff
@@ -123,11 +127,15 @@ impl RangeChecker for Range {
...
 fn taxo_kind_from_ozw_vid(vid: &ValueID) -> Option<ChannelKind> {
     match (vid.get_type(), vid.get_command_class(), vid.get_index()) {
-        (ValueType::ValueType_Bool, Some(CommandClass::DoorLock),     0) => Some(ChannelKind::DoorLocked),
-        (ValueType::ValueType_Bool, Some(CommandClass::SensorBinary), _) => Some(ChannelKind::OpenClosed),
+        (ValueType::ValueType_Bool, Some(CommandClass::DoorLock), 0) => {
+            Some(ChannelKind::DoorLocked)
+        }
+        (ValueType::ValueType_Bool, Some(CommandClass::SensorBinary), _) => {
+            Some(ChannelKind::OpenClosed)
+        }
```
Comment (julienw) [inline components/openzwave-adapter/src/lib.rs]: I could argue that it's less readable after the change.

File: components/openzwave-adapter/src/lib.rs [hunk 2]
```diff
@@ -161,16 +181,22 @@ fn set_ozw_vid_from_taxo_value(vid: &ValueID, value: Value) -> Result<(), TaxoEr
...
             }
         }
         _ => { return Err(TaxoError::InternalError(InternalError::GenericError(format!("Unsupported OZW type: {:?}", vid.get_type())))) }
     };
 
-    result.map_err(|e| TaxoError::InternalError(InternalError::GenericError(format!("Error while setting a value: {}", e))))
+    result.map_err(|e| {
+        TaxoError::InternalError(InternalError::GenericError(format!("Error while setting a \
+                                                                      value: {}",
+                                                                     e)))
```
Comment (julienw) [inline components/openzwave-adapter/src/lib.rs]: here as well

File: components/openzwave-adapter/src/lib.rs [hunk 3]
```diff
@@ -262,94 +294,106 @@ impl OpenzwaveAdapter {
...
                         node_map.push(service_id.clone(), node);
 
                         let mut service = Service::empty(service_id.clone(), adapter_id.clone());
                         service.properties.insert(String::from("name"), node.get_name());
-                        service.properties.insert(String::from("product_name"), node.get_product_name());
-                        service.properties.insert(String::from("manufacturer_name"), node.get_manufacturer_name());
+                        service.properties
+                            .insert(String::from("product_name"), node.get_product_name());
+                        service.properties.insert(String::from("manufacturer_name"),
+                                                  node.get_manufacturer_name());
```
Comment (julienw) [inline components/openzwave-adapter/src/lib.rs:404]: why doesn't the tool do the same thing in the 2 previous lines ?

File: components/openzwave-adapter/src/lib.rs [hunk 4]
```diff
@@ -262,94 +294,106 @@ impl OpenzwaveAdapter {
...
                         });
                     }
-                    ZWaveNotification::NodeNaming(_node)             => {
+                    ZWaveNotification::NodeNaming(_node) => {
                         // unfortunately we can't change a service' properties :(
                         // https://github.com/fxbox/taxonomy/issues/97
                         // When it's done we can move the properties change from above to here.
                     }
-                    ZWaveNotification::NodeRemoved(_node)           => {}
-                    ZWaveNotification::ValueAdded(vid)              => {
```
Comment (julienw) [inline components/openzwave-adapter/src/lib.rs:379]: we tried to have everything aligned here :(

File: components/openzwave-adapter/src/lib.rs [hunk 5]
```diff
@@ -360,44 +404,68 @@ impl OpenzwaveAdapter {
                         };
 
                         for &(ref range, ref sender) in &watchers {
-                            debug!("[OpenzwaveAdapter::ValueChanged] Iterating over watcher {:?} {:?}", taxo_id, range);
+                            debug!("[OpenzwaveAdapter::ValueChanged] Iterating over watcher {:?} \
+                                    {:?}",
```
Comment (julienw) [inline components/openzwave-adapter/src/lib.rs:502]: not sure this is good for readability :(

File: components/openzwave-adapter/src/lib.rs (hunk 3 above)
Comment (fabricedesre) [inline components/openzwave-adapter/src/lib.rs:404]: Not sure, but maybe because of the line length.

**Pull request merged**